notesapi/v1/views.py

```diff
@@ -58,6 +60,8 @@ class AnnotationSearchView(GenericAPIView):
 
         GET /api/v1/search/
         GET /api/v1/search/?course_id={course_id}&user={user_id}
+        GET /api/v1/search/?course_id={course_id}&user={user_id}&&usage_id={usage_id}
+        GET /api/v1/search/?course_id={course_id}&user={user_id}&&usage_id=[{usage_id},{usage_id} ... ]
```

Comment:
I would go for other mechanism i.e. 
`GET /api/v1/search/?course_id={course_id}&user={user_id}&usage_id={usage_id}&id={usage_id} ...`